### DISCUSSION TITLE
Disable alpha / stencil of letters - 3D text with visible background plane?

### DISCUSSION THREAD

**Discussion by thatsmaik** [protectwise/troika#184] [Q&A]
Hello Everyone,

Thanks a lot for this awesome tool-kit!

I'm using troika-three-text and I would like the text to be displayed with a visible background plane instead of the cut-out glyphs. Is it possible to modify the shader in a way that it won't cut-out the glyphs so I can get a visible background rect for each word or letter? Ideal with the possiblity to change the color of the rect.

Thanks in advance for any help!

**Comment by lojjic** [comment]
I suppose it could be done in the same shader, but typically I've just used a separate plane object placed behind the text. If you really want separate rects for each word or character I guess I'd need to know more about your exact use case because there could be a lot of variability in parameters (e.g. how much vertical/horizontal padding, how to handle overlaps, etc.)
